mixins/auth-config.js

```diff
@@ -130,10 +141,6 @@ export default {
         await this.reloadModel();
         this.isEnabling = false;
         btnCb(true);
-        if ( wasEnabled ) {
```

Comment:
On /c/local/auth/config/keycloak?mode=edit` the user isn't taken back to the original view. I think adding `this.editConfig = false;` will fix this`

Reply:
You're completely right; added